task02/task02/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task02
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int a;
+            Console.WriteLine("Введите значение стороны a");
+            a = int.Parse(Console.ReadLine());
+            int b;
+            Console.WriteLine("Введите значение стороны b");
+            b = int.Parse(Console.ReadLine());
+            int gradus;
+            Console.WriteLine("Введите значение угла между ними (в градусах)");
+            gradus = int.Parse(Console.ReadLine());
```

Comment:
Неверно выбран формат данных. Длины и углы не обязательно целые числа.
```suggestion
           double a;
            Console.WriteLine("Введите длину стороны a");
            a = double.Parse(Console.ReadLine());
            double b;
            Console.WriteLine("Введите длину стороны b");
            b = double.Parse(Console.ReadLine());
            double gradus;
            Console.WriteLine("Введите угол между ними (в градусах)");
            gradus = double.Parse(Console.ReadLine());
```